sdcm/sct_events/operator.py

```diff
@@ -58,11 +60,21 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
                 microsecond=int(milliseconds), tzinfo=datetime.timezone.utc).timestamp()
         except Exception:  # pylint: disable=broad-except
             self.timestamp = time.time()
+
+        self.node = str(node)
+        self.line = line
+        self.line_number = line_number
+
+        self._ready_to_publish = True
+
         return self
 
     @property
     def msgfmt(self):
-        return super().msgfmt + ": {0.source} {0.message}"
+        message = super().msgfmt
+        if not self.line:
+            message = message + ": {0.source} {0.message}"
+        return message
```

Comment:
Why do you need this change?

Reply:
`source` and `message` are extracted from the line. If the line is not empty, it will be printed in the message. So `source` and `message` are duplicate of the line and they don't need to be there.

I prefer to print full line from a 2 reasons:
1. it's easier to find the line in the log during investigation
2. in other events (like, NodetoolEvent, Stress events) we print full line and not part of it. I think it's better for analysing

Reply:
It looks like a dead code: if `line` is empty then `source` and `message` are empty too and vice versa. Maybe you need to assign `self.line` before the parsing code then.